uber/templates/groups/form.html

```diff
@@ -183,13 +183,57 @@
             <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
         </div>
     </div>
```

Comment:
This suppresses these form fields in cases where the leader doesn't already have an address filled out.  Won't that make it impossible for an admin to enter this information if it's not already entered?